modules/tinymce/src/themes/silver/main/ts/ui/skin/Loader.ts

```diff
@@ -29,9 +30,8 @@ const loadShadowDomUiSkins = (editor: Editor, skinUrl: string): Promise<void> =>
   }
 };
 
-const loadSkin = (isInline: boolean, editor: Editor): Promise<void> => {
+const loadUrlSkin = (isInline: boolean, editor: Editor): Promise<void | [void, void]> => {
```

Comment:
You could stick an empty `.then()` on the end to replace the one you removed and avoid adding the second promise type

Reply:
Sure, if that's preferrable.